browsers/edge.json

```diff
@@ -56,9 +56,26 @@
         "79": {
           "release_date": "2020-01-15",
           "release_notes": "https://blogs.windows.com/windowsexperience/2020/01/15/new-year-new-browser-the-new-microsoft-edge-is-out-of-preview-and-now-available-for-download/",
-          "status": "current",
+          "status": "retired",
           "engine": "Blink",
           "engine_version": "79"
+        },
+        "80": {
+          "release_date": "2020-02-07",
+          "release_notes": "https://twitter.com/MSEdgeDev/status/1225905079774937088",
```

Comment:
I don't think this qualifies as release notes. :-)

Reply:
I'll go ahead and remove it then!  (This was literally the closest thing I could find to release notes to be honest.  😛)